jwst/regtest/test_miri_lrs_slit_spec2.py

```diff
@@ -30,15 +30,16 @@ def run_pipeline(jail, rtdata_module):
             "--save_bsub=true",
             "--steps.assign_wcs.save_results=true",
             "--steps.flat_field.save_results=true",
-            "--steps.srctype.save_results=true"]
+            "--steps.srctype.save_results=true",
+            "--steps.bkg_subtract.save_combined_background=true"]
```

Comment:
While you'r at it, could you delete the `"--steps.resample_spec.skip=true",` in this test and remove the comment above it about there being a bug.  But is fixed and results look good.

I've already added the truth file to artifactory, so it should run without problem, though you might see small precision differences when you run it locally.

Reply:
Yeah, I wasn't sure if your comment above should be done here or elsewhere, but I can throw it in for good measure. I haven't added the truth file for the saved background, but hoping to okify that, if that works... 😬

Reply:
I can walk you through adding it over slack if you like.